server/server.go

```diff
@@ -3246,6 +3293,45 @@ func (s *StanServer) processSubscriptionRequest(m *nats.Msg) {
 	s.subStartCh <- &subStartInfo{c: c, sub: sub, qs: qs, isDurable: isDurable}
 }
 
+func traceSubscriptionRequest(log logger.Logger, sr *pb.SubscriptionRequest,
```

Comment:
Probably useful to add a unit test for this debug trace string. Could be added into existing tests.